src/common/README.md

```diff
@@ -1,22 +1,14 @@
-# DAOS Common Libraries
-
-Common functionality and infrastructure shared across all DAOS components are provided in external shared libraries. This includes the following features:
+# DAOS Common Libraries Common functionality and infrastructure shared across all DAOS components are provided in external shared libraries. This includes the following features:
```

Comment:
Is it intentional to move the first line to the header?   Actually, did you mean to change these at all?

Reply:
No, that was a search/replace that went wrong. Thanks for catching it. Reverted the whole file except the new section.